sdk/src/CrashReporter.ts

```diff
@@ -208,17 +258,16 @@ export default class CrashReporter {
     const symbolicateStackTrace = require('react-native/Libraries/Core/Devtools/symbolicateStackTrace');
     const cleanedStackFrames: StackFrame[] = __DEV__
       ? await symbolicateStackTrace(stackFrames)
-      : { stack: cleanFilePath(stackFrames) };
-
-    const stack = cleanedStackFrames || [].filter(filterOutReactFrames).map(noAddressAt);
-    setCurrentTags(getCurrentTags().concat('UnhandledError'));
```

Comment:
Looks like the `UnhandledError` tag is no longer being added? Is this intentional?